components/content/table.tsx

```diff
@@ -0,0 +1,76 @@
+interface TableProps {
+  children: JSX.Element | JSX.Element[]
+}
+
+export default function Table({ children }: TableProps): JSX.Element {
+  children = Array.isArray(children) ? children : [children]
+
+  const headings: JSX.Element[] = []
+  const rows: JSX.Element[] = []
+
+  children.map(child => {
+    switch (child.type) {
+      case TableHeading:
+        headings.push(child)
+        break;
+
+      default:
+        rows.push(child)
+        break;
+    }
+  })
+
+  return (
+    <table className="govuk-table lbh-table">
```

Comment:
Nice on creating a reusable table. Note there is also a variant with a caption included:
https://design-system.hackney.gov.uk/components/table#table-with-head-and-caption